.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,42 @@
+name: Ubuntu 18.04 Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+# This allows a subsequently queued workflow run to interrupt previous runs.
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  ubuntu_ci:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Adding HHVM repo
+        run: |
+          curl -fsSL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x058341c68fc8de6017d775a1b4112585d386eb94" | gpg --dearmor > hhvm-archive-keyring.gpg
+          sudo mv hhvm-archive-keyring.gpg /usr/share/keyrings
+          echo "deb [arch=amd64 signed-by=/usr/share/keyrings/hhvm-archive-keyring.gpg] https://dl.hhvm.com/ubuntu bionic main" | sudo tee /etc/apt/sources.list.d/hhvm.list
+      - name: Adding Bazel repo
+        run: |
+          curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+          sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
+          echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+      - name: Installing deps
+        run: |
+          sudo apt update
+          sudo apt install hhvm bazel
+      - name: Checking out
+        uses: actions/checkout@v3
+      - name: Running the type checker
+        run: hh_client .
+      - name: Running tests
+        # TODO: Replace those with //... once //:typechecker_test is fixed.
```

Comment:
There is a comment about typechecker_test being broken in hh.bzl so I substituted this for calling `hh_client` above.